lib/keychain.js

```diff
@@ -140,7 +165,7 @@ export default class Keychain {
         derivationPathLength
       );
       var { byte: curveID, pos: pos } = readByte(binary, pos, 1);
-      var { byte: hashAlgoID } = readByte(binary, pos, 1);
+      var { byte: hashAlgoID, pos: pos } = readByte(binary, pos, 1);
```

Comment:
why do you need this `pos` variable ?

Reply:
it is a fix when you have multiple services (cursor still need to move)

Reply:
ok